graylog2-web-interface/src/components/search/MessageFields.jsx

```diff
@@ -19,7 +19,7 @@ const MessageFields = React.createClass({
 
   _formatFields(fields, showDecoration) {
     if (!showDecoration || !this.props.message.decoration_stats) {
-      return Object.keys(fields).sort().map(key => <MessageField {...this.props} fieldName={key} value={fields[key]} />);
+      return Object.keys(fields).sort().map(key => <MessageField key={key} {...this.props} fieldName={key} value={String(fields[key])} />);
```

Comment:
I don't know why we should do it in `MessageField`, I guess the best place would be in `MessageFieldDescription`, which is the component rendering the field and should be the one knowing how to render the field value.